src/components/DashboardStatistic.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div class="dashboard-statistic">
+    <span class="dashboard-statistic__value">{{ value.toLocaleString() }}</span>
+    <template v-if="isDefined(label)">
+      <span class="dashboard-statistic__label">{{ label.toLocaleString() }}</span>
+    </template>
+    <template v-if="isDefined(meta)">
+      <span class="dashboard-statistic__meta">{{ meta.toLocaleString() }}</span>
+    </template>
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { isDefined } from '@prefecthq/prefect-design'
+
+  defineProps<{
+    value: string | number,
+    label?: string,
+    meta?: string,
+  }>()
+</script>
+
+<style>
+.dashboard-statistic { @apply
+  inline-flex
+  items-end
+  gap-1
+  text-sm
+}
+
+.dashboard-statistic__value { @apply
+  font-semibold
+}
+
+.dashboard-statistic__meta { @apply
+  text-slate-500
```

Comment:
```suggestion
  text-foreground-200
```
This is usually better if you're looking for subdued text in all color modes

Reply:
@brandonreid ^

Reply:
I’m definitely open minded with these details. The card boarder radiuses that I implemented were a good notch or two less than in the design files. Over all I’d advise sticking with conventions as much as possible or feeling free to use your best judgment. Design files don’t always translate to the flexible world of the browser.

When we have a fuller picture we can totally review as a team and find these happy mediums and sim to deploy them system wide. At some point, there’s certainly an initiative around arresting our design tokens from tailwind and making them semantic in a way where we don’t have to share personal mental models, but for now that’s the best we got and we should stick to it.


I’d go with @znicholasbrown’s suggestion for now. If it still seems off though, I can totally take on an audit.